improver/nowcasting/pysteps_advection.py

```diff
@@ -36,6 +36,7 @@
 from iris.coords import AuxCoord
 
 from improver.metadata.amend import (amend_attributes, set_history_attribute)
+from improver.metadata.utilities import generate_mandatory_attributes
```

Comment:
Should the call to `redirect_stdout` be removed (on line 49), given that the `redirect_stdout` function has been removed?

Reply:
The function hasn't actually been removed, it looks like it was accidentally duplicated when the change to get rid of `lib/improver` was merged in.  I just removed the old duplicate.